app/views/issues/_issues_list_view.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="search-box container-fluid">
+  <%= form_with url: filter_issues_path, method: :get do |form| %>
+    <div class="row align-items-center">
+      <div class="col-xl-9 container-fluid row align-items-center">
+        <div class='form-group my-1 float-left'>
+          <%= form.label :status %>
+          <%= form.select(:status, options_for_select([[t('issues.filter.categories.all'), '']] + create_select_options_for_enum(:status, Issue.statuses.keys)), {}, { class: 'mx-2' }) %>
```

Comment:
[[t('issues.filter.categories.all'), '']] + create_select_options_for_enum(:status, Issue.statuses.keys) 
move to me method, do for other filters as well

Reply:
done